test/index.js

```diff
@@ -1,22 +1,38 @@
 var assert = require('assert');
-var Cookies = require('js-cookie');
 var mock = require('mock-cookie');
 
-require('../src/index.js');
-
 document = new mock.Document();
 
 describe('Cookies', function () {
   describe('#removeAll()', function () {
     it('should remove all cookies', function () {
+      var Cookies = require('../src/index.js');
+
       Cookies.set('a', '1');
       Cookies.set('b', '2');
       assert.equal('1', Cookies.get('a'));
       assert.equal('2', Cookies.get('b'));
 
       Cookies.removeAll();
-      assert.equal(undefined, Cookies.get('a'));
-      assert.equal(undefined, Cookies.get('b'));
+
+      var actual = Cookies.get('a');
+      var expected = undefined;
+      assert.strictEqual(actual, expected);
+
+      actual = Cookies.get('a');
+      expected = undefined;
+      assert.strictEqual(actual, expected);
+    });
+    it('should change the original Cookies API', function() {
+      var Cookies = require('js-cookie');
+      require('../src/index.js');
+      assert.ok(!!Cookies.removeAll);
+    });
+    it('should make the API available for a new Cookies.withConverter() instance', function () {
+      var Cookies = require('../src/index.js');
+      var newInstance = Cookies.withConverter();
```

Comment:
The `removeAll` extended method need to be available for new instances of the api created with the `withConverter()` call.